MentorBook.Business/Interfaces/IFriendshipService.cs

```diff
@@ -0,0 +1,11 @@
+using MentorBook.Data.Models;
+using MentorBook.Data.Repositories;
+using System.Collections.Generic;
+ 
+namespace MentorBook.Business
+{
+    public interface IFriendshipService
```

Comment:
It's good practice to have the interface and the concrete implementation with same name i.e. IUserFriendshipService UserFriendshipService.